graphics/fpga-graphics/verilator/main_beam.cpp

```diff
@@ -39,19 +39,24 @@ int main(int argc, char* argv[]) {
         return 1;
     }
 
-    sdl_renderer = SDL_CreateRenderer(sdl_window, -1, SDL_RENDERER_ACCELERATED);
+    sdl_renderer = SDL_CreateRenderer(sdl_window, -1, SDL_RENDERER_ACCELERATED | SDL_RENDERER_PRESENTVSYNC);
```

Comment:
`vsync` is needed to throttle the rendering, as well as make the animation look much smoother. Without `vsync`, it would render at ~100 FPS.